### PR TITLE
Even odd

### PR DESCRIPTION
**Pull request by AliShah51215**
Activity 5 Even Odd Numbers

### CODE REVIEW

File: src/main/java/Age/Age.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package Age;
+
+import static Age.AgeTest.VerifyAge;
+
+class Main {
+    public static void main (String[]args){
+        System.out.println("Are you an Adult? " +VerifyAge(16));
```
Comment (mtszpater) [inline src/main/java/Age/Age.java:7]: czemu to zadanie jest w tym pull requescie?

File: src/main/java/Age/AgeTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package Age;
+
+public class AgeTest {
+
+    public static boolean VerifyAge(int age) {
```
Comment (mtszpater) [inline src/main/java/Age/AgeTest.java:5]: czemu to zadanie jest w tym pull requescie?

File: src/test/java/Age/AgeTestTest.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AgeTestTest {
+
+    @Test
+    void shouldReturnFalseWhenAgeIs13() {
+
+        Boolean result = AgeTest.VerifyAge(13);
```
Comment (mtszpater) [inline src/test/java/Age/AgeTestTest.java:12]: czemu to zadanie jest w tym pull requescie?

**Review by AliShah51215 — COMMENTED** [review]
file deleted, there was error during push what caused issue

**Pull request closed without merging**